acceptance/bundle/override/job_tasks/output.txt

```diff
@@ -1,5 +1,6 @@
 {
   "name": "job",
+  "permissions": [],
```

Comment:
Why did this entry appear?

Reply:
Because ApplyBundlePermissions is done before we validate remote paths

Reply:
So we've changed order for static resources as well --- that's expected I guess. However, I still don't understand why does changing order changes output of "validate -o json" on static resources?

Reply:
@denik I'm also not sure if it has any effects, what about enabling recording requests, doing a deploy and see if there are any new permission related calls made?

Reply:
Is it being deployed? As I see command is failing:
```
>>> errcode [CLI] bundle validate -o json -t staging
Error: file test1.py not found
```

Reply:
We reordered the application of bundle permissions and validation for paths. That's why when validation fails, we see bundle with permissions applied